### PR TITLE
Disable money effect in pause #5733

### PR DESCRIPTION
**Pull request by RMcTn**

### CODE REVIEW

**Comment by rwjuk** [comment]
Your first three commits are unrelated, can you get rid of those (rebase the last one on the latest commit from the develop branch).

**Comment by Gymnasiast** [comment]
You'll need to rebase your branch.

Also, not entirely sure if this is the best place to put the check, but I'd like to hear from @IntelOrca first.

File: src/openrct2/game.c [hunk 1]
```diff
@@ -596,8 +596,10 @@ sint32 game_do_command_p(sint32 command, sint32 *eax, sint32 *ebx, sint32 *ecx,
                 finance_payment(cost, gCommandExpenditureType);
                 if (gUnk141F568 == gUnk13CA740) {
                     // Create a +/- money text effect
-                    if (cost != 0)
-                        money_effect_create(cost);
+                    if (cost != 0) {
+                        if (game_is_not_paused())
```
Comment (rwjuk) [inline src/openrct2/game.c]: You can just say `if(cost !=0 && game_is_not_paused())`

**Comment by Gymnasiast** [comment]
This is worse... I can do it for you, if you want me to.

**Comment by rwjuk** [comment]
That's better! 👍

**Comment by RMcTn** [comment]
Yeah sorry haha, still trying to get used to git!

**Pull request merged**